packages/wdio-appium-service/tests/launcher.test.js

```diff
@@ -83,8 +84,12 @@ describe('Appium launcher', () => {
 
             expect(launcher.process).toBeInstanceOf(MockProcess)
             expect(launcher.logPath).toBe('./')
-            expect(launcher.command).toBe('path/to/my_custom_appium')
-            expect(launcher.appiumArgs).toMatchSnapshot()
+            if(isWindows){
```

Comment:
@mgrybyk  the failure is caused by the warnings thrown from obsolete snapshots. I have used a condition to validate snapshot only when the os is not windows. Also, we have separate snapshot test for windows. We don't have a way to remove these warnings. Only way is to use -u flag which updates snapshots everytime and passes automatically which is not a right way to do it. I am thinking to remove match snapshot from these tests and write separate snapshot tests for mac & linux like windows. Let me know your thoughts.

Reply:
Separate snapshots should work, it's up to you if you have a better idea